backend/app/services/internal/auth_service.py

```
@@ -57,20 +60,27 @@ def login(self, login_data: LoginRequest) -> dict:
         """Update user's password."""
         if not verify_password(old_password, user.password_hash):
             raise UnauthorizedException(message="Invalid old password")
         if new_password == old_password:
-            raise BadRequestException(message="New password cannot be the same as the old password")
-            
+            raise BadRequestException(
+                message="New password cannot be the same as the old password"
+            )
+
         new_password_hash = get_password_hash(new_password)
         updated_user = self.user_repo.update_password(user.id, new_password_hash)
```

Ruff (0.14.13)

Local variable `updated_user` is assigned to but never used

Remove assignment to unused variable `updated_user`

(F841)